translations/Arabic/special-terms.md

```diff
@@ -0,0 +1,48 @@
+# Terms
+
+- Message: رسالة
```

Comment:
I did it just like this document: https://github.com/zulip/zulip-gci/blob/master/tasks/translation-terms.md
It had only one line, but it's fine, I will add another